ehr/resources/views/ehrAdmin.html

```diff
@@ -1,96 +1,44 @@
 <script type="text/javascript">
 
-Ext4.onReady(function (){
+</script>
 
-if (LABKEY.Security.currentUser.isAdmin){
-    var webpart = <%=webpartContext%>;
+<html>
+<div class="col-lg-3">
+<div class="lead">
+    <h4><b>Admin:</b></h4>
+    <div><a class="labkey-text-link" href='<%=contextPath%><%=containerPath%>/study-manageStudy.view?'>Manage EHR Study</a></div>
```

Comment:
I know this syntax works, but I always prefer to use double quotes in HTML for attributes.
```suggestion
    <div><a class="labkey-text-link" href="<%=contextPath%><%=containerPath%>/study-manageStudy.view?">Manage EHR Study</a></div>
```

Reply:
Updated